test/xUnit/Asserts/PriorityAttribute.cs

```diff
@@ -3,13 +3,16 @@
 
 using System;
 
-[AttributeUsage(AttributeTargets.Method, AllowMultiple = false)]
-public class TestPriorityAttribute : Attribute
+namespace PSTests.Internal
 {
-    public TestPriorityAttribute(int priority)
+    [AttributeUsage(AttributeTargets.Method, AllowMultiple = false)]
+    public class PriorityAttribute : Attribute
```

Comment:
Why do we rename the attribute? The change is not related to the PR.

Reply:
The `Test` prefix on the attribute appeared unnecessary now it is declared inside the `PSTests` namespace.